checker/tests/nullness/java8inference/FalsePositives.java

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
https://github.com/typetools/checker-framework/pull/1387#issuecomment-316147360 for other parts.